Mackbellemore/soen-390-team07
# feat(client & server): Production Page

#

## Changes this PR introduces

- Production Page UI
- Production End Point
- Production Controller Unit Test



## How to test this PR

- Test End Points
- Make sure the production page renders
- Make sure the production displays values




https://share.clickup.com/t/h/nq08fz/WNC6I6C01LP5JNB



## Notes for reviewers

- 🥂



## Checklist
